### DISCUSSION TITLE
[Script request] NX Witness

### DISCUSSION THREAD

**Discussion by HeynekeWeb** [community-scripts/ProxmoxVE#364] [Request script] [answered]
### Application Name

NX Witness

### Website

https://nxvms.com/download/releases/linux

### Description

NX Witness is a selfhosted NVR with a Mobile App & various other supported devices intergartions.
Nx Witness hosts a diverse network of partners who leverage Nx for their security solutions or seamlessly integrate with our comprehensive toolkit.
It offers remote monitoring and cloud storage.
I installed mine on a Unbuntu Container and it works wonders. 

The resources I allowcated to mine are:
CPU Count: 4
RAM: 4GB
Storage: 32GB (I'm only using 5GB there of with 6 Cameras)

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by aldekmt** [comment]
+1 would like to see this as script, maybe as lxc based on ubuntu 22.04

**Comment by Mati-l33t** [comment]
Looks interesting. Do you need some kind of license for this or is it free and opensource? What kind of cpu usage are you getting with 6 cameras?

**Comment by tremor021** [comment, marked as answer]
#2350 Done here
